commitserver/commit/hydratorhelper.go

```diff
@@ -137,6 +143,27 @@ func writeReadme(root *os.Root, dirPath string, metadata hydratorMetadataFile) e
 	return nil
 }
 
+func writeGitAttributes(root *os.Root, dirPath string) error {
+	gitAttributesPath := filepath.Join(dirPath, ".gitattributes")
+	gitAttributesFile, err := root.Create(gitAttributesPath)
+	if err != nil && !os.IsExist(err) {
+		return fmt.Errorf("failed to create git attributes file: %w", err)
+	}
+
+	fileContents := "*/README.md linguist-generated=true\n*/hydrator.metadata linguist-generated=true\n"
```

Comment:
Maybe pull this to the top of the file as a constant? I'd use a string literal (backticks) for readability.